config.yaml

```diff
@@ -0,0 +1,11 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+options:
+  update-time-range:
+    type: string
+    description: >
+      A preferred update time in 24 hour format in UTC for the Jenkins LTS patch update to take
```

Comment:
Keep in mind that we should aim to keep the grammar in the docs as simple as possible. Also, try not to use Latin terms over English ones. Some general guidelines can be found here https://developers.google.com/style/abbreviations. I wasn't able to find a topic covering this case in our documentation.

How about something in the lines of  `Preferred time for LTS patch updates in 24 hour UTC format. If empty, the update will take place whenever an update is available. For example, 03-05 will allow updates to take place from 3AM UTC to 5AM UTC.`

Reply:
Updated the description, thank you for the guidelines!